ttnn/cpp/ttnn/operations/eltwise/unary/device/unary_device_operation.cpp

```diff
@@ -137,7 +137,7 @@ void UnaryDeviceOperation::validate_on_program_cache_miss(
 
     if (preallocated_output_tensor.has_value()) {
         const auto compited_output_shape = compute_output_shapes(args, tensor_args);
```

Comment:
Typo?

Reply:
That is done in the next PR https://github.com/tenstorrent/tt-metal/pull/15092